addons/package_hierarchy/models/package_links.py

```diff
@@ -52,7 +52,7 @@ class PackageHierarchyLink(models.Model):
         ondelete="cascade",
         check_company=True,
     )
-    company_id = fields.Many2one("res.company", default=False)
+    company_id = fields.Many2one("res.company", default=lambda self: self.env.company)
```

Comment:
Super nit: Can we add an explicit string for `Company`? I think the Odoo default would be `Compand id`, unless it is smarter with fields ending in `_id`?

I know we don't show it on a view, but it would still show on an Advanced Search for example.